definitions/checks/disk/performance.rb

```diff
@@ -1,9 +1,13 @@
 module Checks
   module Disk
     class Performance < ForemanMaintain::Check
+      DEFAULT_DIRS = [
+        '/var/lib/pulp', '/var/lib/mongodb', '/var/lib/pgsql'
+      ].select { |file_path| File.directory?(file_path) }.freeze
+
       metadata do
         label :disk_performance
-        description 'Check for recommended disk speed of pulp, mongodb, pgsql dir.'
+        description "Check recommended disk speed for #{DEFAULT_DIRS.join(',')} directories."
```

Comment:
This line will be too long and break the output. Maybe if we modify the def above to 
```ruby
DEFAULT_DIRS = %i[pulp mongo foreman_database].inject({}) do |dirs, f| 
  dirs[feature(f).label_dashed] = feature(f).data_dir if feature(f) && File.directory?(feature(f).data_dir)
  dirs
end
```
we could use something like `DEFAULT_DIRS.keys.join(', ')` to print the services we are going to test.
What do you think?